heareval/predictions/task_predictions.py

```diff
@@ -79,6 +82,117 @@ def __getitem__(self, idx):
         # return self.embedding_memmap[idx], self.labels[idx]
 
 
+def create_events_from_prediction(
+    prediction_dict: Dict, threshold: float = 0.5
```

Comment:
Maybe since we expect the key to be float timestamp, etc. put that in the docstring and type here?

Reply:
Like Dict[float, float]

Reply:
I still am not 100% sure what the signature is for prediction dict. Does it involve the label or index? Is there are probability?

Reply:
Updated this : Dict[float, torch.Tensor]